tests/html5skin-test.js

```diff
@@ -521,6 +521,69 @@ describe('Controller', function() {
       expect(controller.state.controlBarVisible).toBe(true);
     });
 
+    describe('test customPlayPause', () => {
+      let spy;
+      let testEvent;
+      let testPlayerState;
+      const onTogglePlayPause = (event, playerState) => {
+        testEvent = event;
+        testPlayerState = playerState;
+      };
+
+      beforeEach(function() {
+        spy = sinon.spy(controller.mb, 'publish');
```

Comment:
as we use jest maybe replace it with `jest.fn()`?

Reply:
\+